water/lib/Dialect/Wave/Transforms/LowerReadWriteOps.cpp

```diff
@@ -57,51 +61,70 @@ make1DTransferCommonAttrs(MemRefType memrefType, int64_t vectorizedDim,
 /// Materialize affine.apply for expressions inside a `map` with `symbols`.
 /// Each symbol is either a GPU id (thread/block) or a constant from `hyper`.
 static FailureOr<SmallVector<Value>>
-materializeAffine(Location loc, ArrayRef<wave::WaveSymbolAttr> symbols,
-                  AffineMap map, PatternRewriter &rewriter,
+materializeAffine(Location loc, ArrayRef<Attribute> symbols, AffineMap map,
+                  PatternRewriter &rewriter,
                   wave::WaveHyperparameterAttr hyper) {
   // NOTE: This helper assumes 0 dims in `map`. If you add dims, prepend
   // the dim operands before the symbol operands below.
   assert(map.getNumDims() == 0 && "expected 0 dims");
 
   auto threadId = [&](gpu::Dimension d) -> Value {
-    return rewriter.create<gpu::ThreadIdOp>(loc, rewriter.getIndexType(), d);
+    return gpu::ThreadIdOp::create(rewriter, loc, rewriter.getIndexType(), d);
   };
   auto blockId = [&](gpu::Dimension d) -> Value {
-    return rewriter.create<gpu::BlockIdOp>(loc, rewriter.getIndexType(), d);
+    return gpu::BlockIdOp::create(rewriter, loc, rewriter.getIndexType(), d);
```

Comment:
I'd rather not have irrelevant changes in the PR. This is going to be harmful when we need to investigate/bisect errors some time later.